app/stable_appconfig.go

```diff
@@ -49,60 +45,23 @@ func setCustomKVStoreKeys() []string {
 }
 
 func (app *RegenApp) registerUpgradeHandlers() {
+
 	// This is the upgrade plan name we used in the gov proposal.
-	upgradeName := "v2.0-upgrade"
-	app.UpgradeKeeper.SetUpgradeHandler(upgradeName, func(ctx sdk.Context, plan upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
-		// 1st-time running in-store migrations, using 1 as fromVersion to
-		// avoid running InitGenesis.
-		// Explicitly skipping x/auth migrations. It is already patched in regen-ledger v1.0.
-		fromVM := map[string]uint64{
-			"auth":         auth.AppModule{}.ConsensusVersion(),
-			"bank":         1,
-			"capability":   1,
-			"crisis":       1,
-			"distribution": 1,
-			"evidence":     1,
-			"gov":          1,
-			"mint":         1,
-			"params":       1,
-			"slashing":     1,
-			"staking":      1,
-			"upgrade":      1,
-			"vesting":      1,
-			"ibc":          1,
-			"genutil":      1,
-			"transfer":     1,
-			"ecocredit":    1, // we don't run InitGenesis for ecocredit in `RunMigrations`, but manually instead.
+	const upgradeName = "v3.0.0"
+	app.UpgradeKeeper.SetUpgradeHandler(upgradeName, func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		// TODO: once integrating basepp need to register a proper ecocredit consensus version
+		// fromVM[ecocredit.ModuleName] = ecocreditmodule.Module{}.ConsensusVersion() - 1
+
+		ecoParams, ok := app.ParamsKeeper.GetSubspace(ecocredit.ModuleName)
+		if !ok {
+			return nil, fmt.Errorf("unable to upgrade: subspace %s not found", ecocredit.ModuleName)
 		}
 
-		gen := ecocredittypes.DefaultGenesisState()
-		gen.Params.AllowlistEnabled = true
-		gen.Params.CreditClassFee = sdk.NewCoins(sdk.NewCoin("uregen", ecocredittypes.DefaultCreditClassFeeTokens))
-
-		modules := make(map[string]json.RawMessage)
-		modules[ecocredittypes.ModuleName] = app.cdc.MustMarshalJSON(gen)
-		app.smm.InitGenesis(ctx, modules, []abci.ValidatorUpdate{})
-
-		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
-	})
+		// set basket creation fee to 1'0000 REGEN
```

Comment:
```suggestion
		// set basket creation fee to 1,000 REGEN
```

Reply:
Personally I prefer `'` because it's more clear. In France and Spain, `,` is used for decimal separator.

Reply:
maybe just not separate it at all?

Reply:
Let's be consistent and use `,` as we do elsewhere in our codebase.

Reply:
We have been traditionally using `.` for decimals and `,` for long numbers.

Reply:
Agreed that `,` is more consistent with our usage elsewhere.